happypose_ros/happypose_ros/utils.py

```diff
@@ -218,12 +242,70 @@ def get_camera_transform(
     :rtype: geometry_msgs.msg.TransformStamped
     """
 
-    pose_vec = pin.SE3ToXYZQUAT(pin.SE3(camera_pose.numpy()))
     return TransformStamped(
         header=header,
         child_frame_id=child_frame_id,
-        transform=Transform(
-            translation=Vector3(**dict(zip("xyz", pose_vec[:3]))),
-            rotation=Quaternion(**dict(zip("xyzw", pose_vec[3:]))),
-        ),
+        transform=transform_mat_to_msg(camera_pose.numpy()),
+    )
+
+
+def continuous_symmetry_to_msg(
+    symmetry: happypose_symmetries.ContinuousSymmetry,
+) -> ContinuousSymmetry:
+    """Converts HappyPose ContinuousSymmetry object into ContinuousSymmetry ROS message
+
+    :param symmetry: HappyPose object storing definition of continuous symmetry.
+    :type symmetry: happypose_symmetries.ContinuousSymmetry
+    :return: ROS message representing continuous symmetry.
+    :rtype: happypose_msgs.msg.ContinuousSymmetry
+    """
+    return ContinuousSymmetry(
+        # Explicit conversion to float is required in this case
```

Comment:
why?

Reply:
I added more explanation in the comment. Should be clearer now